test/OpenTelemetry.Tests/Resources/ResourceTest.cs

```diff
@@ -129,28 +133,41 @@ public void CreateResource_SupportedAttributeTypes()
             var attributes = new Dictionary<string, object>
             {
                 { "string", "stringValue" },
-                { "long", 1L },
                 { "bool", true },
                 { "double", 0.1d },
+                { "long", 1L },
 
                 // int and float supported by conversion to long and double
                 { "int", 1 },
```

Comment:
if u can, add int[], short[], float[] and we will see if that would be long[], long[] and double[]

Reply:
In any current scenario, these throw an exception since the SanitizeValue() method in Resource.cs at cs:108.
We already help convert int, float, and short to the supported types, should we do the same for arrays?